src/app/main/component/events/components/images-container/images-container.component.ts

```diff
@@ -70,8 +95,18 @@ export class ImagesContainerComponent implements OnInit {
     this.images.splice(i, 1);
     this.imgArray.splice(i, 1);
     this.imgArrayOutput.emit(this.imgArray);
-    this.images.push({ src: null, label: this.dragAndDropLabel, isLabel: false });
+    if (this.editMode) {
+      this.deleteImagesOutput.emit(this.imagesTodelete);
+    }
+    let allowLabel = false;
+    if (i === this.imagesCount - 1) {
```

Comment:
remove if
let allowLabel = i === this.imagesCount - 1;

Reply:
@hnativlyubomyr Hi, I got some issues with Github, this change is not related to this pr.